scripts/wrappers/start.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -eu
+
+
+usage() {
+cat << EOF
+usage: start.sh --init-security yes --admin-password ...
+To be ran / setup once per cluster.
+--host (Optional)               Host specification (default: localhost:5601)
+--port (Optional)               Port specification (default: localhost:5601)
+--opensearch <uri1>,<uri2>      Opensearch instance URIs
+--help                          Shows help menu
+EOF
+}
+
+
+# Args
+# The host/IP where where the dashboard is to be exposed
+host=""
+# Port where the dashboard is to be exposed
+port=""
+# The same as the opensearch.hosts configuration parameter
+opensearch=""
+
+
+# Args handling
+function parse_args () {
+    host="$(snapctl get host)"
+    port="$(snapctl get port)"
+    opensearch="$(snapctl get opensearch)"
+}
+
+function set_defaults () {
+    if [ -z "${host}" ]; then
+        host="localhost"
+    fi
```

Comment:
can be oneliner: `host=${host:=localhost}`
or for the case above:
```
[ -n "${opensearch:-}" ] || opensearch="https://localhost:9200"
```

Reply:
One question regarding the above.

Why is the `:-` (setting the default to "") is important in the `test -n` query?
It seemed to me that if the variable is unset, the check still completes as expected but I may be overlooking or missing something.

Reply:
TL;DR: You can ignore this, it was my muscle memory on the review. :-)

Long:  actually the modern [google shell style says](https://google.github.io/styleguide/shellguide.html#features-and-bugs) your way is correct!